File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    let answer = [];
    const len=commands.length;
    
    for(let i=0; i<len; i++){
        let Array=[...array].slice(commands[i][0]-1, commands[i][1]).sort((a,b)=>a-b);
        console.log(Array);
        answer.push(Array[commands[i][2]-1]);
    }
    return answer;
}

File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.*;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        for(int i=0; i<commands.length; i++){
            answer[i]=func(array, commands[i]);
        }
        return answer;
    }
    
    public static int func(int[] array, int[] command){
        int[] sliced=Arrays.copyOfRange(array, command[0]-1, command[1]);
        Arrays.sort(sliced);
        return sliced[command[2]-1];
    }
}